### PR TITLE
Update publications.yml

### PR DESCRIPTION
**Pull request by ngsrinivas**
Added a SIGCOMM and SOSR publication. Kindly lemme know if it's okay to add these. thanks!

### CODE REVIEW

File: _data/publications.yml [hunk 1]
```diff
@@ -110,3 +110,17 @@
...
+-
+    title: "Heavy-hitter detection entirely in the data plane"
+    authors: "Vibhaalakshmi Sivaraman, Srinivas Narayana, Ori Rottenstreich, S. Muthukrishnan, and Jennifer Rexford "
+    venue: "ACM SIGCOMM Symposium on Software-defined Networking Research"
+    date: '2017-03-01'
+    url: 'https://dl.acm.org/citation.cfm?id=3063772'
+    type: 'conference'
+-
+    title: "Language-directed hardware design for network performance monitoring"
+    authors: "Srinivas Narayana, Anirudh Sivaraman, Vikram Nathan, Mohammad Alizadeh, David Walker, Jennifer Rexford, Vimalkumar Jeyakumar, and Changhoon Kim "
```
Comment (jnfoster) [inline _data/publications.yml]: This author list doesn't match up with the published version. Please fix?

File: _data/publications.yml [hunk 2]
```diff
@@ -110,3 +110,17 @@
...
     type: 'workshop'
+-
+    title: "Heavy-hitter detection entirely in the data plane"
+    authors: "Vibhaalakshmi Sivaraman, Srinivas Narayana, Ori Rottenstreich, S. Muthukrishnan, and Jennifer Rexford "
+    venue: "ACM SIGCOMM Symposium on Software-defined Networking Research"
+    date: '2017-03-01'
+    url: 'https://dl.acm.org/citation.cfm?id=3063772'
+    type: 'conference'
+-
+    title: "Language-directed hardware design for network performance monitoring"
```
Comment (jnfoster) [inline _data/publications.yml]: Please capitalize like the paper?

File: _data/publications.yml [hunk 3]
```diff
@@ -110,3 +110,17 @@
     date: '2018-08-20'
     url: 'https://conferences.sigcomm.org/sigcomm/2018/workshop-netcompute.html'
     type: 'workshop'
+-
+    title: "Heavy-hitter detection entirely in the data plane"
```
Comment (jnfoster) [inline _data/publications.yml]: Please capitalize like the paper?

**Comment by ngsrinivas** [comment]
Hi Nate, thanks for the comments. Please lemme know if anything looks amiss now (it's my first time working with comments on pull requests)

File: _data/publications.yml [hunk 4]
```diff
@@ -68,20 +68,34 @@
...
     title: "Swing State: Consistent Updates for Stateful and Programmable Data Planes"
     authors: "Shouxi Luo, Hongfang Yu, Laurent Vanbever"
     venue: "ACM SIGCOMM Symposium on Software Defined Networking Research (SOSR)"
     date: '2017-04-03'
     url: 'https://dl.acm.org/citation.cfm?id=3050233'
     type: 'conference'
+-
+    title: "Language-Directed Hardware Design for Network Performance Monitoring"
+    authors: "Srinivas Narayana, Anirudh Sivaraman, Vikram Nathan, Prateesh Goyal, Venkat Arun, Mohammad Alizadeh, Vimalkumar Jeyakumar, and Changhoon Kim"
+    venue: "ACM SIGCOMM"
```
Comment (jnfoster) [inline _data/publications.yml]: Please edit this line so it has the full name of the conference with the acronym in parenthesis, like the other entries.

File: _data/publications.yml [hunk 5]
```diff
@@ -68,20 +68,34 @@
     date: '2017-04-03'
     url: 'http://doi.acm.org/10.1145/3050220.3050228'
     type: 'conference'        
+-
+    title: "Heavy-Hitter Detection Entirely in the Data Plane"
+    authors: "Vibhaalakshmi Sivaraman, Srinivas Narayana, Ori Rottenstreich, S. Muthukrishnan, and Jennifer Rexford "
+    venue: "ACM SIGCOMM Symposium on Software-defined Networking Research"
```
Comment (jnfoster) [inline _data/publications.yml]: Please edit this line so it has the full name of the conference with the acronym in parenthesis, like the other entries.

File: _data/publications.yml (hunk 4 above)
Comment (ngsrinivas) [reply _data/publications.yml]: Hi Nate, how should I enter a full name for the ACM SIGCOMM conference? (I couldn't find any other entries here on the main sigcomm conference; only CCR)

https://conferences.sigcomm.org/sigcomm/2017/

Comment (jnfoster) [reply _data/publications.yml]: We've been going with: "ACM Conference of the Special Interest Group on Data Communication (SIGCOMM)"

**Comment by jnfoster** [comment]
Looks great. Thank you!

**Pull request merged**